src/apps/Settings/Settings.js

```diff
@@ -126,116 +110,113 @@ class Settings extends React.Component {
-                      }
-                      return `Please unlock your account in MetaMask.`
-                    })()}
-                  </Text>
-                )}
-              </div>
+      <AppLayout title="Settings">
+        <Content>
+          <Option
+            name="Test Tokens"
+            text="Deposit some tokens into your organization for testing purposes."
+          >
+            <div>
+              <Button
+                mode="secondary"
+                onClick={this.handleDepositTestTokens}
+                disabled={!enableTransactions}
+              >
+                Request Test Tokens
+              </Button>
+              {!enableTransactions && (
+                <Text size="small" style={{ marginLeft: '10px' }}>
+                  {(() => {
+                    if (userNetwork !== network.type) {
+                      return `Please select the ${
+                        network.type
+                      } network in MetaMask.`
+                    }
+                    return `Please unlock your account in MetaMask.`
+                  })()}
+                </Text>
+              )}
+            </div>
+            <Note>
+              Requesting tokens will assign random <strong>TEST</strong> tokens
+              to your organization. The tokens are named after existing
+              projects, but keep in mind <strong>THEY ARE NOT</strong> the real
+              ones.{' '}
+              <span role="img" aria-label="winking face">
+                😉
+              </span>{' '}
+              You can view the received tokens in the Token Balances on the
+              Finance app.
+            </Note>
+          </Option>
+          <Option
+            name="Organization Address"
+            text={`This organization is deployed on the ${network.name}.`}
+          >
+            <Field label="Address">
+              <FieldTwoParts>
+                <TextInput readOnly wide value={daoAddr} />
+                <EtherscanLink address={daoAddr}>
+                  {url =>
+                    url ? (
+                      <LinkButton href={url} target="_blank">
+                        See on Etherscan
+                      </LinkButton>
+                    ) : null
+                  }
+                </EtherscanLink>
+              </FieldTwoParts>
               <Note>
-                Requesting tokens will assign random <strong>TEST</strong>{' '}
-                tokens to your organization. The tokens are named after existing
-                projects, but keep in mind <strong>THEY ARE NOT</strong> the
-                real ones.{' '}
-                <span role="img" aria-label="winking face">
-                  😉
-                </span>{' '}
-                You can view the received tokens in the Token Balances on the
-                Finance app.
+                <strong>Do not send Ether to this address!</strong> Use the
+                Finance app address below for sending{' '}
```

Comment:
I'll just comment to say that we should make this more explicit as "Please send Ether to the Finance app address below."

Reply:
I agree. As nothing will happen if people actually do send it, as it will revert